src/main/java/com/shippingcostestimator/enterprise/dto/ShippingCost.java

```diff
@@ -15,7 +15,7 @@
 public @Data
 class ShippingCost {
     int totalCostId;
-    int serviceLevel;
+    int carrierServiceLevel;
```

Comment:
https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c#:~:text=How%20meaningful%20%3F,does%20not%20reveal%20its%20intent.

Its good to be as clear as you can with variable names. So, I just changes one to add some clarity.